__init__.py

```diff
@@ -15,7 +15,7 @@
 
 __version__ = paddleocr.VERSION
 __all__ = [
-    'PaddleOCR', 'PPStructure', 'draw_ocr', 'draw_structure_result',
+    'PaddleOCR', 'PPStructure', 'PPStructureX', 'draw_ocr', 'draw_structure_result',
```

Comment:
Can `PPStructure` be modified directly instead of creating a new class? I prefer to have only one `PPStructure`.

Reply:
Agreed, I'll got it done soon!

Reply:
Done.